README.md

```diff
@@ -167,6 +167,14 @@ Note that execution options are applied lazily - on the `execute()` method call,
 
 ReadOnly/ReadWrite mode of a connection can't be changed while a transaction is in progress - first you must commit or rollback it.
 
+### Stale reads 
+To use the Spanner [Stale Reads](https://cloud.google.com/spanner/docs/reads#perform-stale-read) with SQLAlchemy you can tweak the connection execution options with a wanted staleness value. For example:
```

Comment:
Read timestamp and minimum read timestamp expect a `datetime.datetime` object.
Maximum staleness and exact staleness expect a `datetime.timedelta` object.
This should probably be documented here in some way, possibly with examples for each, WDYT?

Reply:
Okay, added examples for all of them